src/CONST.ts

```diff
@@ -5109,6 +5116,42 @@ const CONST = {
                 },
             ],
         },
+        [onboardingChoices.TRACK_WORKSPACE]: {
+            message: 'Here are some important tasks to help get your workspace set up.',
+            video: {
+                url: `${CLOUDFRONT_URL}/videos/guided-setup-manage-team-v2.mp4`,
+                thumbnailUrl: `${CLOUDFRONT_URL}/images/guided-setup-manage-team.jpg`,
+                duration: 55,
+                width: 1280,
+                height: 960,
+            },
+            tasks: [
+                createWorkspaceTask,
+                meetGuideTask,
+                setupCategoriesTask,
+                {
+                    type: 'inviteAccountant',
+                    autoCompleted: false,
+                    title: 'Invite your accountant',
+                    description: ({workspaceMembersLink}) =>
+                        '*Invite your accountant to Expensify and share your expenses with them to make tax time easier.\n' +
```

Comment:
```suggestion
                        '*Invite your accountant* to Expensify and share your expenses with them to make tax time easier.\n' +
```

I can see this is exactly copied from the design doc but the trailing * for the bold is missing.

Reply:
@ishpaul777 This is still not fixed.